sources/packages/web/src/services/http/InstitutionApi.ts

```diff
@@ -74,17 +74,15 @@ export class InstitutionApi extends HttpBaseClient {
     );
   }
 
-  async getInstitutionTypeOptions(): Promise<OptionItemDto[]> {
-    try {
-      const response = await this.apiClient.get(
-        "institution/type/options-list",
-        this.addAuthHeader(),
-      );
-      return response.data;
-    } catch (error) {
-      this.handleRequestError(error);
-      throw error;
-    }
+  /**
+   * Get the list os all institutions types to be returned in an option
```

Comment:
Typo here. Is this supposed to be "Get the list of all institutions types" ?

Reply:
It was already fixed based on @ann-aot comment 😉